Xamarin.Essentials/Maps/Map.ios.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+using MapKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Map
+    {
+        internal static Task PlatformOpenMapsAsync(double latitude, double longitude, MapLaunchOptions options)
+        {
+            if (string.IsNullOrWhiteSpace(options.Name))
+                options.Name = string.Empty;
+
+            NSDictionary dictionary = null;
+            var placemark = new MKPlacemark(new CLLocationCoordinate2D(latitude, longitude), dictionary);
+            return OpenPlacemark(placemark, options);
+        }
+
+        internal static async Task PlatformOpenMapsAsync(Placemark placemark, MapLaunchOptions options)
+        {
+            var adress = new MKPlacemarkAddress
```

Comment:
should make this an extension method as it would be re-usable.

Reply:
I would suggest doing so, but in a separate pull request. Our placemark type is not very intuitive and has no sample adress. This would definitely be advisable to minimise risk from misuse by different developers